src/constant/global.ts

```diff
@@ -106,6 +106,7 @@ const apiStatusCodes = {
 
 const IN_DEV_PAGES = ['/projects'];
 const projectGroupWhatsapp = 'https://chat.whatsapp.com/D1ko12SykD1LfvJwmNQ48A';
+const admins = ['helloatkrishna@gmail.com', 'krish36pandey@gmail.com'];
```

Comment:
don't do it like this. use admin password. if password correct then ok else no. redo with password